atlp-rwanda/elite-bn-backend
# fix tripComments routes && documentation

#### What does this PR do?
> Documentation of travel-request comments routes
#### Description of Task to be completed
> Fixing the tripComments routes
> Fix the route for getting all comments on the travel request
> Documentation of comment routes
#### How should this be manually tested?
> Clone https://github.com/atlp-rwanda/elite-bn-backend
>run npm install
> run server `npm run dev`
> run our api docs `http://localhost:5000/api-docs/` , Sign in
> Check the comment routes and execute tests
#### Any background context you want to provide?
> N/A
#### What are the relevant Trello story?
> https://trello.com/c/mBd7C8O0/73-comment-on-travel-requests
#### Screenshots (if appropriate)
> N/A
